docs/getting_started/03_intro_level_3.ipynb

```diff
@@ -221,7 +221,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.13"
+   "version": "3.10.11"
```

Comment:
These types of changes ares probably going to be an unfortunate side effect of multiple people (with multiple versions of python) editing these files... for now, I'd suggest we manually override those changes for now (so as not to end up in a vicious circle) 


```suggestion
   "version": "3.7.13"
```